lmdeploy/model.py

```diff
@@ -543,7 +543,11 @@ def messages2prompt(self, messages, sequence_start=True, tools=None, **kwargs):
             if role == 'assistant' and message.get('tool_calls', None) is not None:
                 for tool_call in message['tool_calls']:
                     function = tool_call.get('function', {})
-                    function['arguments'] = function.pop('parameters', {})
+                    function['name'] = function.get('name', '')
+                    function['parameters'] = function.get('arguments', function.get('arguments', ''))
```

Comment:
why two `function.get('arguments')`

Reply:
Oh, it is a typo. It should be `parameters`.